content/tab-programming.html

```diff
@@ -496,19 +561,23 @@ <h3 class="fi">Vertailuoperaattorit</h3>
 </pre>
 
 <p class="fi">Loogiset konnektiivit:</p>
+<p class="sv">Logiska konnektiv:</p>
 
 <pre>
-and		ja
-or		tai
-not		ei
+and		ja		ja
```

Comment:
Mitkä ovat ohjelmoinnin ruotsissa käytety sanat loogisille operaatioille AND, OR ja NOT?
Pitäisikö AND olla OCH ja NOT olla ICKE?

Reply:
Hieno korjaus!